benchmark/buffers/buffer-copy.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  size: [10, 1024, 2048, 4096, 8192],
+  n: [1024]
+});
+
+function main({ n, size }) {
+  const source = Buffer.allocUnsafe(size);
+  const target = Buffer.allocUnsafe(size);
+
+  bench.start();
+  for (var i = 0; i < n * 1024; i++) {
+    source.copy(target);
```

Comment:
It would be great to test some options as well to see how the `sourceStart` / `sourceEnd` etc. compete :)